src/utils.ts

```diff
@@ -218,6 +223,11 @@ export function safeLink(addr: string, config: Config): string {
   return explorerLink(addr, config);
 }
 
+// Splits a string at a defined index.
+export function splitAt(input: string, index: number): [string, string] {
+  return [input.slice(0, index), input.slice(index + 1)];
+}
```

Comment:
This function skips a character, is that intended?
```
splitAt("abcdef", 3)
[ "abc", "ef" ]
```

Reply:
Yes, this skips a character because we don't want the path to start with a "/" and I thought better do it here..
But maybe to maintain the logic of the function we could move that to `splitPrefixFromPath`

Reply:
Hmm, but this function is exported as a general utility. Can we instead embed it inside the function that uses it?